### DISCUSSION TITLE
Can't send test message

### DISCUSSION THREAD

**Discussion by alexk1919** [postalserver/postal#2958] [Help with using Postal]
I have everything setup according. to the installation guide.

postal.yml:
version: 2

postal:
  web_hostname: post.example.com
  web_protocol: https
  smtp_hostname: post.example.com

main_db:
  host: 127.0.0.1
  username: root
  password: postal
  database: postal

message_db:
  host: 127.0.0.1
  username: root
  password: postal
  prefix: postal

smtp_server:
  default_bind_address: "::"
  port: 25
  tls_enabled: true

dns:
  # Specify the DNS records that you have configured. Refer to the documentation at
  # https://github.com/atech/postal/wiki/Domains-&-DNS-Configuration for further
  # information about these.
  mx_records:
    - mx.post.example.com
  spf_include: spf.post.example.com
  return_path_domain: rp.post.example.com
  route_domain: routes.post.example.com
  track_domain: track.post.example.com

smtp:
  # Specify an SMTP server that can be used to send messages from the Postal management
  # system to users. You can configure this to use a Postal mail server once the
  # your installation has been set up.
  host: 127.0.0.1
  port: 2525
  username: # Complete when Postal is running and you can
  password: # generate the credentials within the interface.
  from_name: Postal
  from_address: postal@post.example.com

rails:
  # This is generated automatically by the config initialization. It should be a random
  # string unique to your installation.
  secret_key: 240f49c4b0e4254fbf
  
  
  Can someone help?

**Comment by willpower232** [comment]
Have you actually filled in the `smtp:` section? Particularly the username and password?

**Reply by alexk1919** [reply]
What do I set that to?

**Reply by willpower232** [reply]
> You can configure this to use a Postal mail server once ~the~ your installation has been set up

**Reply by alexk1919** [reply]
Thanks. The problem was a far more basic issue... my VPS is blocking port 25.
